infrastructure/terraform/per_workspace/modules/api_entrypoint/vars.tf

```diff
@@ -15,3 +15,6 @@ variable "project" {
 
 variable "authoriser_metadata" {
 }
+
+variable "domain" {
+}
```

Comment:
Very small one - sometimes type is specified and sometimes it's not? Is this just a preference?

Reply:
I think this is probably an inconsistency thing on our part, most of the ones I look at that do have a type have differed from that of a string so we should probably align everything soon to at least include a type haha